js/filter-set.js

```diff
@@ -6,8 +6,6 @@ var URI = require('urijs');
 
 var Filter = require('./filters').Filter;
 
-var KEYCODE_ENTER = 13;
```

Comment:
I can't recall, but I think it's because we used to have a handler for keydown that was using that keycode. The var isn't referenced anymore in the file though, so I deleted it.